dashboard/config/scripts/csp_unit_1_question_1.evaluation_question

```diff
@@ -0,0 +1,5 @@
+name 'CSP Unit 1 Question 1'
+question 'What is the hardest thing about teaching computer science?'
+answer 'Getting my students to write', 1, 'Strategies for Student Writing'
```

Comment:
Yeah, I think named parameters here would make it clearer and be a bit more flexible.